tests/phpunit/tests/Styles/TestHooks.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace WPML\PB\Cornerstone\Styles;
+
+use WPML\LIB\WP\PostMock;
+
+/**
+ * @group styles
+ */
+class TestHooks extends \OTGS_TestCase {
+
+	use PostMock;
+
+	public function setUp() {
+		parent::setUp();
+		$this->setUpPostMock();
+	}
+
+	/**
+	 * @test
+	 */
+	public function itShouldAddHooks() {
+		$subject = $this->getSubject();
+
+		add_action( 'save_post', [ $subject, 'invalidateStylesInTranslation' ] );
```

Comment:
@strategio Shouldn't this be `expectActionAdded`

Reply:
Good catch, corrected!